README.md

```diff
@@ -17,7 +17,7 @@ A common library to abstract the Amazon SQS and SNS producers/consumers interact
 Add the package to your dependencies and the bintray resolver.
 
 ```sbtshell
-libraryDependencies += "com.taxis99" %% "common-sqs" % "0.3.0"
+libraryDependencies += "com.taxis99" %% "common-sqs" % "0.3.0-RC1"
```

Comment:
The `RCX` versions are meant for "advanced" usage only, don't think this should be placed in the `README` as default.